src/components/molecules/ChatList/ChatMessage.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useCallback } from "react";
+
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+
+import { WithId } from "utils/id";
+import { formatUtcSeconds } from "utils/time";
+import { getLinkFromText } from "utils/getLinkFromText";
+
+import { useUser } from "hooks/useUser";
+
+import { User } from "types/User";
+
+import {
+  PrivateChatMessage,
+  RestrictedChatMessage,
+} from "components/context/ChatContext";
+
+interface ChatMessageProps {
+  message: WithId<RestrictedChatMessage | PrivateChatMessage>;
+  usersById: Record<string, User>;
+  allowDelete: boolean;
+  onAvatarClick: (
+    message: WithId<RestrictedChatMessage | PrivateChatMessage>
+  ) => void;
+  onDeleteClick: (
+    message: WithId<RestrictedChatMessage | PrivateChatMessage>
+  ) => void;
+}
+
+export const ChatMessage: FC<ChatMessageProps> = ({
+  message,
+  usersById,
+  allowDelete,
+  onAvatarClick,
+  onDeleteClick,
+}) => {
+  const { user } = useUser();
+
+  const profileImageSize = 30;
```

Comment:
[NB, nit] since this is static, it could be declared outside of the component.

No need to worry about it for this PR though